shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -295,9 +295,13 @@ - (void)resetChannels {
 - (void)setupChannels {
   // This will be invoked once the shell is done setting up and the isolate ID
   // for the UI isolate is available.
+  __block fml::WeakPtr<FlutterEngine> weakSelf = [self getWeakPtr];
```

Comment:
The __block specified does nothing for fml::WeakPtr.

Reply:
Done.